src/main/resources/assets/computercraft/lua/rom/apis/parallel.lua

```diff
@@ -31,36 +49,82 @@ local function runUntilLimit( _routines, _limit )
                     else
                         tFilters[r] = param
                     end
-                    if coroutine.status( r ) == "dead" then
-                        _routines[n] = nil
-                        living = living - 1
-                        if living <= _limit then
-                            return n
+                    if _data then
```

Comment:
Instead of doing this whole `_data` argument and resizing the count based on that, I do wonder if it'd be better to do the "any/all" flag as discussed elsewhere. That way it's pretty much just: when all: "return when a coroutine dies", when any: "return when the live list is empty".

Should drop the whole need for the `_data` parameter, which is a tad ugly.

Reply:
I guess it makes sense to do that now, since the functionality to stop running outside of any/all has been implemented. Will do.